literature/briega-martos_2021_cation_48/briega-martos_2021_cation_48_p2_f1_Cs_2.3.yaml

```diff
@@ -10,62 +8,62 @@ electrochemical system:
       material: Pt
       shape: coiled wire
     reference electrode:
-      source:
-        supplier: none
       type: RHE
     working electrode:
-      crystallographic orientation: 111
+      crystallographic orientation: '111'
       geometric electrolyte contact area:
-        unit: cm-2
-        value: null
+        unit: 1 / cm2
       material: Pt
-      preparation procedure: null
       shape:
         diameter:
           unit: mm
           value: 2
         height:
           unit: mm
-          value: null
         type: bead
-      source:
-        LOT: null
-        supplier: null
   electrolyte:
     components:
     - concentration:
         unit: mol / l
         value: 0.1
       name: CsF
       source:
-        supplier: Alfa Aesar, Puratronic 99.99% (metal basis)
-      sum formula: CsF
+        supplier: Alfa Aesar Puratronic
+        supplied purity:
+          grade:
+            value: 99.99
+            unit: pct
       type: salt
     - name: water
       source:
-        comment: deoxigenated H2O2 solution (Fluka, TraceSELECT Ultra, for trace analysis).
         quality: ultrapure water
-      sum formula: H2O
       type: solvent
     - concentration:
         unit: mol / l
-        value: null
-      name: perchloric acid
+      name: HClO4
       source:
-        LOT: null
         supplier: Merck
-      sum formula: HClO4
       type: acid
     ph:
-      uncertainty: null
       value: 2.3
     temperature:
       unit: K
       value: 298.15
     type: aq
 source:
-  bib: briega-martos_2021_cation_XXX
+  citation key: briega-martos_2021_cation_48
   curve: First cycle
   url: https://doi.org/10.1021/acsmeasuresciau.1c00004
   figure: 1
 version: 1
+curation:
+  version: 1
+  process:
+  - role: curator
+    name: Nicolas Hoermann
+    orcid: https://orcid.org/0000-0001-6944-5575
+    date: 2021-09-21
```

Comment:
In the back of my mind: Wasn't there a problem with date not given as string?
See line 69 below.

Reply:
The issue was related to JSON, which does not understand datetime. Hence datetime has to be converted to a string, before it can be dumped to JSON.